backend/routes/auth.py

```diff
@@ -1,112 +1,100 @@
-from flask import redirect
-from flask import request
 from flask import Blueprint
-from flask_login import login_required
-from flask_login import login_user
-from flask_login import logout_user
 from ..database import db
-from ..database.models.User import User
-from ..database.models.User import login_manager
-from ..database.models.User import user_manager
+from ..database import User, UserRole
+from ..database import login_manager
+from ..auth import role_required
+from flask_jwt_extended import (
+    create_access_token,
+    get_jwt_identity,
+    jwt_required,
+)
+from ..auth import user_manager
+from ..schemas import UserSchema
+from ..dto import RegisterUserDTO, LoginUserDTO
+from flask_pydantic import validate
 
 
-bp = Blueprint("auth", __name__)
+bp = Blueprint("auth", __name__, url_prefix="/api/v1/auth")
 
 
+# TODO: Place cookie on users browser with JWT token
```

Comment:
Consider using flask_login for browser sessions and JWT's for programmatic access. flask_login handles setting/refreshing cookies for you. If not, we should remove the flask_login stuff in user.py

Reply:
I can start working on that. I think we could make that a new issue and PR, so that this one doesn't get too much larger